apps/studio/src/schemas/page.ts

```diff
@@ -13,6 +13,34 @@ export const getEditPageSchema = z.object({
   siteId: z.number().min(1),
 })
 
+export const reorderBlobSchema = z.object({
+  pageId: z.number().min(1),
+  from: z.number().min(0),
+  to: z.number().min(0),
+  siteId: z.number().min(0),
+  blocks: z.array(
+    z
+      .object({
+        type: z.string(),
+      })
+      .passthrough(),
+  ),
+})
+
+export const updatePageSchema = getEditPageSchema.extend({
```

Comment:
Why is siteId required for updatePageBlobSchema but not here?

Reply:
actually idt we need for both but @karrui added on the `pageBlob` but not here

Reply:
I think we should standardise to query with siteId + resourceId

Reply:
this has `siteId` - `getEditPageSchema` has `siteId + pageId`

Reply:
agree should be both right. each page has pageId and siteId. Even though we know that those ids are all unique, we should still pass it to the server (for future authorisation checks)

Reply:
yeap, i'm fine with having both. i was thinking we could do the retrieval implicitly (ie, if you query for resource A with siteId 2, we assume your `siteId = 2`) but i think having it so that resource authorisation checks can be simplified as middleware makes sense.

my point above was that the schema itslef has `siteId` because it's extending a schema with `pageId + siteId`
<img width="605" alt="image" src="https://github.com/user-attachments/assets/2365e6ac-b3f5-4348-bf2a-f5aa9d0a0843">

Reply:
One comment here is that the siteId should come from the user's auth context which defines what sites they have access to, and not from user's input